src/js/components/Columns.js

```diff
@@ -104,13 +105,24 @@ export default class Columns extends Component {
   _getColumnBreakpoints () {
     // grab CSS styles from DOM after component mounted
     // default to small size ($size-small = 192px)
-    let minColumnWidth = 192;
     const container = findDOMNode(this.containerRef);
     const column = container.childNodes[0];
     const child = column.childNodes[0];
+    let minColumnWidth = 192;
+    let childStyles;
 
     if (child) {
-      const childStyles = window.getComputedStyle(child);
+      if (navigator && navigator.userAgent.indexOf('MSIE') !== -1
```

Comment:
The 10ms timeout was only needed for IE11, but I've updated the timeout to get applied in all cases. After further testing, I found that Firefox wasn't getting the correct child widths in time (window.getComputedStyle), and required a 200ms delay.

For now, I've updated the timeout to use 200ms, but I'm wondering if it would be better to set the styles based on hardcoded widths, instead of searching for the DOM element and grabbing its width style property (which is not always set in time).